src/libopenrave-core/generictrajectory.cpp

```diff
@@ -217,7 +217,7 @@ class GenericTrajectory : public TrajectoryBase
         return it1->second < it2->second;
     }
 
-    void Init(const ConfigurationSpecification& spec) override
+    void Init(const ConfigurationSpecification& spec, const int nWayPointsToReserve=0, const int option=0) override
```

Comment:
@snozawa Can we add an enum to this option?

Reply:
sure!
I added the enum.
Also, update the docstring for `CloningOptions` to guide to use the existing enum.